src/gossip/service.zig

```diff
@@ -1942,9 +1942,9 @@ pub const GossipService = struct {
 
 /// stats that we publish to prometheus
 pub const GossipStats = struct {
-    gossip_packets_received: *Counter,
-    gossip_packets_verified: *Counter,
-    gossip_packets_processed: *Counter,
+    gossip_packets_received_total: *Counter,
```

Comment:
Isn't "total" implied by it being a counter? I feel like you could add "total" to the name of every counter and it would be just as accurate.

Reply:
> Isn't "total" implied by it being a counter? I feel like you could add "total" to the name of every counter and it would be just as accurate.

I agree, but I did this because Grafana was complaining about using a variable not ending in `_total` for `rate(`.

Reply:
<img width="411" alt="image" src="https://github.com/user-attachments/assets/d0379fd3-f172-4683-9235-78485ae7a8a1">

Reply:
`_total` ending is recommended by the [Prometheus team](https://prometheus.io/docs/practices/naming) itself

Reply:
Fair enough. I can see the reasoning for using this name when exposed to prometheus and grafana, since the http API does not include type information.

Maybe we should open another pr to rename all counters, and make this a standard practice.